src/components/plot/bookmarkbutton.tsx

```diff
@@ -42,7 +42,7 @@ export class BookmarkButtonBase extends React.PureComponent<BookmarkProps, Bookm
       >
         <i
           className="fa fa-bookmark command-bookmark"
-          style = {{color: bookmarkColor, paddingLeft: '3px'}}
+          style = {{color: bookmarkColor, paddingLeft: '12px', cursor: 'pointer'}}
```

Comment:
Instead of adding a custom `style`, we should add a class called `command`, which set `margin-left` (not padding), `cursor`, and `color` and use it for all command buttons in `<Plot>`